reactor-netty-http/src/main/java/reactor/netty/http/server/MicrometerHttpServerMetricsRecorder.java

```diff
@@ -114,6 +135,32 @@ public void incrementErrorsCount(SocketAddress remoteAddress, String uri) {
 		}
 	}
 
+	@Override
+	public void incrementActiveConnections(String uri, String method, int amount) {
+		MeterKey meterKey = new MeterKey(uri, null, method, null);
+		Counter activeConnections = activeConnectionsCache.get(meterKey);
+		activeConnections = activeConnections != null ? activeConnections : activeConnectionsCache.computeIfAbsent(meterKey,
+				key -> filter(activeConnectionsBuilder.tags(URI, uri, METHOD, method)
+						.register(REGISTRY)));
+
+		if (activeConnections != null) {
+			activeConnections.increment(amount);
+		}
+	}
+
+	@Override
+	public void incrementServerConnections(SocketAddress localAddress, int amount) {
+		String address = Metrics.formatSocketAddress(localAddress);
+		MeterKey meterKey = new MeterKey(null, address, null, null);
+		Counter connections = connectionsCache.get(meterKey);
+		connections = connections != null ? connections : connectionsCache.computeIfAbsent(meterKey,
+						key -> filter(connectionsBuilder.tags(URI, HTTP, LOCAL_ADDRESS, address).register(REGISTRY)));
+
+		if (connections != null) {
+			connections.increment(amount);
+		}
+	}
```

Comment:
this override is not needed. the super method already does the same thing, since `super(HTTP_SERVER_PREFIX, HTTP)` correctly configures the meter builder name and URI tag. as a result, `connectionsCache` and `connectionsBuilder` are also redundant

```suggestion
```

Reply:
good catch !! 
thanks for the review, I'll work on applying all this.